modules/ding_campaign_plus/ding_campaign_plus.features.field_instance.inc

```diff
@@ -383,7 +419,9 @@ Campaign box is located at the bottom of the left column, on the page types wher
     'widget' => array(
       'active' => 1,
       'module' => 'options',
-      'settings' => array(),
+      'settings' => array(
+        'match_operator' => 'CONTAINS',
+      ),
```

Comment:
I was very confused as to why features was exporting this. Looking at `options_field_widget_info()` from the options module, that defines the option_select widget, it doesn't have any settings.

Instead this setting is actually added to the option_select widget by ding_nodelist module in `ding_nodelist_field_widget_info_alter()`. I think this is really weird. I can't see the setting used anywhere in ding_nodelist or related modules and does it even make sense to have contains match operator on an select field?

@guddo Do you know why this was added? Is it something we can remove to avoid confusion?